cloud-sql/postgres/sqlalchemy/connect_tcp.py

```diff
@@ -0,0 +1,98 @@
+"""
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+db_config = {
```

Comment:
Same here -- can we move this into the body of the function and wrap it with an exclude? Or even just move this below the import statements? Or is it pythonic to do this?